package.json

```diff
@@ -87,8 +88,8 @@
 		"@storybook/addon-storysource": "5.3.2",
 		"@storybook/addon-viewport": "5.3.2",
 		"@storybook/react": "5.3.2",
+		"@testing-library/jest-dom": "5.3.0",
```

Comment:
Is this what gives us extra assertions like `expect( ... ).toHaveAttribute` ? Part of me thinks they're a nice convenience, and there's another part of me that thinks the abstractions are just going to make life more difficult to have a new set of functions to learn, vs. reusing existing knowledge of how to get at DOM values (e.g.  `expect( element.getAttribute( ... ) ).toBe`).

Reply:
Yes, it's that. I can try to revisit assertions and use native DOM code instead 👍

Reply:
6dcba54 – we probably can skip this addition, it took me 5 minutes to rewrite tests :)